Remove all aliases to get/list/create/set/delete commands

cdrage opened the pull request:
This PR removes the `ocdev component X` commands for the following
commands:

```sh
Available Commands:
  application Perform application operations
  catalog     Catalog related operations
  completion  Output shell completion code
  create      Create new component
  delete      Delete existing component
  get         Get currently active component
  help        Help about any command
  list        List all components in the current application
  project     Perform project operations
  push        Push source code to component
  set         Set active component.
  storage     Perform storage operations
  url         Expose component to the outside world
  version     Print the version of ocdev
```

kadel commented on cmd/set.go:
```diff
@@ -0,0 +1,40 @@
+)
+
+var componentSetCmd = &cobra.Command{
+	Use:   "set",
```
This command should be called `component` and should be coupled together with `set`

```
# getting current component
ocdev component

# switching current component 
ocdev component <name>
```

if you look `oc project` in works in the same way

kadel replied:
There will be no `set` command. 
`ocdev component <name>` will act as set. If there is no argument, it will just print current application.
Look at `oc project`, it works in the same way.

cdrage replied:
@kadel Ah, that makes sense. Was a bit confused. I'll update the PR.

kadel replied:
no problem, thank ;-)

cdrage commented:
Hopefully tests pass, but I updated this :+1: I wasn't able to get the e2e tests running locally, so just waiting on TravisCI.

kadel commented on cmd/create.go:
```diff
@@ -1,6 +1,121 @@
+	},
+}
+
 // 'ocdev crate' is just an alias for 'ocdev component create'
```
we can remove this commend

kadel replied:
remove comment

kadel commented on cmd/delete.go:
```diff
@@ -0,0 +1,69 @@
+	},
+}
+
+// 'ocdev list' is just an alias for 'ocdev component list'
```
remove comment

kadel commented on cmd/list.go:
```diff
@@ -1,5 +1,40 @@
+	},
+}
+
 // 'ocdev list' is just an alias for 'ocdev component list'
```
remove comment

cdrage commented:
@kadel 

Re-based, removed the uneeded comments, and also added the default `ocdev component foobar` set command.

```sh
github.com/redhat-developer/ocdev  remove-aliases ✔                                                                                                                                                                                                                         0m  ⍉
▶ ./ocdev component
The current component is: foobar

github.com/redhat-developer/ocdev  remove-aliases ✔                                                                                                                                                                                                                          0m  
▶ ./ocdev component get
The current component is: foobar

github.com/redhat-developer/ocdev  remove-aliases ✔                                                                                                                                                                                                                          0m  
▶ ./ocdev component set foo
Switched to component: foo

github.com/redhat-developer/ocdev  remove-aliases ✔                                                                                                                                                                                                                          0m  
▶ ./ocdev component foobar
Switched to component: foobar

github.com/redhat-developer/ocdev  remove-aliases ✔                                                                                                                                                                                                                          0m  
▶ ./ocdev component
The current component is: foobar
```

Code for the if statement for the set / get command is here: https://github.com/redhat-developer/ocdev/pull/279/files#diff-1dcc122e377b3a7b4adf52faf26c0188R19
